### PR TITLE
show PDF and PPT

### PR DESCRIPTION
**Pull request by Tahia50**

### CODE REVIEW

File: FileReaderApp/ViewController.m [hunk 1]
```diff
@@ -82,7 +86,20 @@ - (void)matchExtension:(NSString *)clickedFileType {
         [FileAlert showAlertWithController:self message:@"Selected File Type Mismatch"];
         return;
     }
-    //push to next Controller
+    QLPreviewController * preview = [[QLPreviewController alloc] init];
```
Comment (hasanmainul) [inline FileReaderApp/ViewController.m]: unnecessary space between pointer and variable name :D

File: FileReaderApp/ViewController.m [hunk 2]
```diff
@@ -82,7 +86,20 @@ - (void)matchExtension:(NSString *)clickedFileType {
         [FileAlert showAlertWithController:self message:@"Selected File Type Mismatch"];
```
Comment (ahammad-reyad) [inline FileReaderApp/ViewController.m]: Move all the string resource to a localizable string file

File: FileReaderApp/ViewController.m [hunk 3]
```diff
@@ -10,12 +10,14 @@
...
 #import "PublicStringDefine.h"
+#import <QuickLook/QuickLook.h>
 
-@interface ViewController ()<DOPDropDownMenuDataSource, DOPDropDownMenuDelegate>
+@interface ViewController ()<DOPDropDownMenuDataSource, DOPDropDownMenuDelegate, QLPreviewControllerDelegate, QLPreviewControllerDataSource>
 
 @property (strong, nonatomic) NSArray *filesArray;
 @property (strong, nonatomic) DOPDropDownMenu *menu;
 @property (copy, nonatomic) NSString *fileName;
+@property (assign, nonatomic) NSInteger theIndex;
```
Comment (dina28) [inline FileReaderApp/ViewController.m]: what is the purpose of theIndex naming?? simple index

File: FileReaderApp/ViewController.m [hunk 4]
```diff
@@ -82,7 +86,20 @@ - (void)matchExtension:(NSString *)clickedFileType {
...
+    preview.delegate = self;
+    preview.currentPreviewItemIndex = self.theIndex;
+    [self.navigationController pushViewController:preview animated:YES];
+}
+
+- (NSInteger)numberOfPreviewItemsInPreviewController:(QLPreviewController *)controller {
+    return self.filesArray.count;
+}
+
+- (id <QLPreviewItem>)previewController: (QLPreviewController *)controller previewItemAtIndex:(NSInteger)index {
```
Comment (dina28) [inline FileReaderApp/ViewController.m]: remove space..

File: FileReaderApp/ViewController.m [hunk 5]
```diff
@@ -10,12 +10,14 @@
...
 #import "FileAlert.h"
 #import "PublicStringDefine.h"
+#import <QuickLook/QuickLook.h>
 
-@interface ViewController ()<DOPDropDownMenuDataSource, DOPDropDownMenuDelegate>
+@interface ViewController ()<DOPDropDownMenuDataSource, DOPDropDownMenuDelegate, QLPreviewControllerDelegate, QLPreviewControllerDataSource>
 
 @property (strong, nonatomic) NSArray *filesArray;
 @property (strong, nonatomic) DOPDropDownMenu *menu;
 @property (copy, nonatomic) NSString *fileName;
```
Comment (jannatulabeda) [inline FileReaderApp/ViewController.m]: Make it strong, don't use copy if not needed

File: FileReaderApp/ViewController.m (hunk 2 above)
Comment (jannatulabeda) [inline FileReaderApp/ViewController.m]: @ahammad-reyad  localizable string file not needed in this project
@Tahia50 but can be moved to a constant file

**Comment by hasanmainul** [comment]
1bf5jzd

**Comment by jannatulabeda** [comment]
Show the attached image as AppIcon
filereader

**Comment by dina28** [comment]
ok 👍

**Pull request merged**